R/available-data.R

```diff
@@ -44,10 +56,6 @@ spod_available_data <- function(
 #' @param xml_url The URL of the XML file to download. Defaults to "https://opendata-movilidad.mitma.es/RSS.xml".
 #'
 #' @return The path to the downloaded XML file.
-#' @examples
-#' if (FALSE) {
```

Comment:
They don't like `if(FALSE)` examples then?

Reply:
Used to be fine with these.

Reply:
> They don't like `if(FALSE)` examples then?

It does seem ugly. But they asked to remove all examples from non exported functions, and all of those were with 'if(FALSE)'. So problem solved)